portfolio/src/main/webapp/css/main.scss

```diff
@@ -0,0 +1,328 @@
+@import url("https://fonts.googleapis.com/css2?family=Poppins:wght@300;400;700&display=swap");
+body {
+  height: 100vh;
+  margin: 0;
+  font-family: 'Poppins', arial;
+}
+
+ul {
+    list-style-type: none;
+    margin: 0;
+    padding: 0;
+}
+
+a {
+    text-decoration: none;
+}
+
+img {
+    width: 100%;
+}
+
+.navigation-container {
+    background: #1616AC;
+
+    a { /* sass exclusive feature */
+        color: white;
+    }
+}
+
+header {
+    display: flex;
+    justify-content: space-between; /* This line makes header columns in two lines*/
+    padding: 2em;
+
+    .logo {
+        font-weight: bold;
+    }
+
+
+
+    ul {
+        display: flex;
+
+        a {
+            display: block;
+            padding: 0 1em;
+        }
+    }
+    
+}
+
+.social-header, .hero-design {
+    display: none; /* This is off due to mobile view*/
+}
+
+.hero {
+    background: #2424A0;
+    color: white;
+    display: grid;
+    text-align: center;
+    padding: 4em;
+}
+
+.meet {
+    display: flex;
+    margin: 0 auto;
+    width: 200px;
+    font-weight: bold;
+    color: #FA824C
+
+}
+
+span {
+    margin-top: 1em;
+    margin-right: .5em;
+}
+
+.scroll {
+    width: 30px;
+    margin-top: 2em;
+
+}
+
+section {
+    padding: 4em 2em;
+    text-align: center;
+}
+  
+.featured {
+    position: relative;
+}
+
+.featured::before {
+    content: "";
```

Comment:
Use single ('') rather than double ("") quotation marks for attribute selectors and property values, see https://google.github.io/styleguide/htmlcssguide.html#CSS_Quotation_Marks.

Reply:
changed the quotes